src/Microsoft.Performance.SDK/Processing/ColumnMetadata.cs

```diff
@@ -155,12 +156,29 @@ public ColumnMetadata(ColumnMetadata other)
         public Guid Guid { get; }
 
         /// <summary>
-        ///     Gets the name of this column. If this column has a dynamic
+        ///     Gets or initializes the name of this column. If this column has a dynamic
         ///     name, then this property always returns the name as based
         ///     on the data in the first row. Use <see cref="NameProjection"/>
         ///     for dynamic names.
```

Comment:
Are there any documents explaining the difference between a column's label and name? Still confused after reading the comments

Reply:
Label's are associated with Column Metadata; the label is only associated with the variant.

I think this will be more clear once we have an implementation in an SDK driver that we can use to augment our documentation. The final docs for this feature are still a WIP, so I'm OK with some things still being a bit ill-defined (especially as we are still developing them).

Reply:
> Label's are associated with Column Metadata

I assume you mean the names are associated with column metadata?

Reply:
yep, my bad!